cli/src/main/kotlin/commands/RequirementsCommand.kt

```diff
@@ -120,7 +122,7 @@ class RequirementsCommand : OrtCommand(
                             val actualVersion = tool.getVersion()
                             runCatching {
                                 val isRequiredVersion = tool.getVersionRequirement().let {
-                                    it == CommandLineTool.ANY_VERSION || it.isSatisfiedBy(actualVersion)
+                                    Semver.coerce(actualVersion).satisfies(it)
```

Comment:
Why is the check for `ANY_VERSION` dropped?

Reply:
`ANY_VERSION` was introduced here in d8c65c3f8ac329fc89c01f2694b03f5cf6d41309 to work around a limitation of the original `isSatisfiedBy()` call to not handle non-semantic versions, but even those should be accepted here. This is not necessary anymore with the new code as it uses `coerce()` before.